annals-of-botany.csl

```diff
@@ -104,14 +104,25 @@
           </group>
           <text prefix=" " suffix="." macro="publisher"/>
         </if>
+        <else-if type="webpage" match="any">
+          <group suffix=". ">
+            <text macro="title" font-style="italic" prefix=" "/>
+          </group>
+          <text variable="URL" prefix=" " suffix="."/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix=" "/>
+             <date-part name="month" form="short" suffix=" "/>
+             <date-part name="year" form="long"/>
+           </date>
+        </else-if>    
         <else-if type="chapter paper-conference" match="any">
           <text macro="title" prefix=" "/>
-          <group prefix=".">
+          <group prefix=" In: ">
             <group>
               <group suffix=".">
-                <names variable="editor translator">
-                  <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always" prefix=" In: " suffix=","/>
-                  <label form="short" prefix=" "/>
+                <names variable="editor translator" delimiter="," suffix=" ">
+                  <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always"/>
+                  <label form="short" prefix=", " suffix=""/>
```

Comment:
@damnation333, it's also cleaner to delete empty prefixes and suffixes, like `suffix=""` here.